src/Volume.ts

```diff
@@ -238,42 +240,63 @@ export default class Volume {
     this.normRegionOffset = subregionOffset.clone().divide(volumeSize);
   }
 
+  /** Returns `true` iff differences between `loadSpec` and `loadSpecRequired` indicate new data *must* be loaded. */
+  private mustLoadNewData(): boolean {
+    return (
+      this.loadSpec.time !== this.loadSpecRequired.time || // time point changed
+      !this.loadSpec.subregion.containsBox(this.loadSpecRequired.subregion) || // new subregion not contained in old
+      this.loadSpecRequired.channels.some((channel) => !this.loadSpec.channels.includes(channel)) // new channel(s)
+    );
+  }
+
+  /**
+   * Returns `true` iff differences between `loadSpec` and `loadSpecRequired` indicate a new load *may* get a
+   * different scale level than is currently loaded.
```

Comment:
Can you say why this is a "maybe" and not "definitely will load new scale level?"

Reply:
Updated JSDoc comment here. It says:

> This checks for changes in properties that *can*, but do not *always*, change the scale level the loader picks. For example, a smaller subregion *may* mean a higher scale level will fit within memory constraints, or it may not. A higher `scaleLevelBias` *may* nudge the volume into a higher scale level, or we may already be at the max imposed by `multiscaleLevel`.